csharp/src/iceboxnet/ServiceManagerI.cs

```diff
@@ -692,8 +692,8 @@ private void stopAll()
             }
 
             //
-            // For each service, we call stop on the service and flush its database environment to
```

Comment:
Odd comment about flushing database environments. An IceBox service may or may not use a database.  And IceBox itself doesn't maintain any database.